stock_picking_product_barcode_report/wizard/stock_barcode_selection_printing.py

```diff
@@ -21,7 +21,6 @@ class ProductPrintingQty(models.TransientModel):
     uom_id = fields.Many2one(
         "uom.uom",
         string="Unit of Measure",
-        related="move_line_id.product_uom_id",
```

Comment:
Is this change contemplated for the rest of the wizard calls or would it be advisable to make this field compute readonly=False?